gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -496,3 +496,9 @@ def add_arguments(self, parser):
               'chromosome is requested. Only variants matching at least one '
               'of these regions will be loaded. If this parameter is not '
               'specified, all variants will be kept.'))
+    parser.add_argument(
+        '--call_names',
+        default=None, nargs='+',
+        help=('A list of call names (separated by a space). Only variant '
```

Comment:
nit: `Only variants for these calls will be ...` and also, s/samples/calls at the end.